framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -91,6 +91,7 @@ namespace bpl {
 #define DEFAULT_BAND_STEERING 1
 #define DEFAULT_DFS_REENTRY 1
 #define DEFAULT_CLIENT_ROAMING 1
+constexpr int DEFAULT_PERSISTENT_DB = 0;
```

Comment:
Comment?

Reply:
added comments to all the persistent-db related `constexpr`